internal/cba.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// Package dca contains utils for implementing Device Certificate
+// cba.go (certificate-based access) contains utils for implementing Device Certificate
 // Authentication according to https://google.aip.dev/auth/4114
```

Comment:
and Default Credentials for Google Cloud Virtual Environments according to https://google.aip.dev/auth/4115.